Makefile

```diff
@@ -1,3 +1,7 @@
+# Makefile
```

Comment:
It looks like you are trying to be consistent with terraform/Makefile here, but can we maybe rather drop this comment in both files?

Reply:
Sure.